src/common/Testimonial/TestimonialModal.jsx

```diff
@@ -40,21 +40,40 @@ export default function TestimonialModal({ isOpen, setIsOpen }) {
     fetchCategories();
   }, []);
 
+  function escapeNewLinesAndQuotes(inputString) {
+    const regex = /[\n\r"']/g;
+    const escapeMap = {
+      '\n': '\\n',
```

Comment:
@yung-coder Are these the only characters to worry about? Then this fix is fine.